config/config.go

```diff
@@ -131,6 +131,7 @@ type Performance struct {
 	StatsLease      string `toml:"stats-lease" json:"stats-lease"`
 	RunAutoAnalyze  bool   `toml:"run-auto-analyze" json:"run-auto-analyze"`
 	StmtCountLimit  int    `toml:"stmt-count-limit" json:"stmt-count-limit"`
+	CommitTimeout   string `toml:"commit-timeout" json:"commit-timeout"`
```

Comment:
Seems more suitable to put it to `[tikv-client]`.

Reply:
ok